src/config/dynamic-theme-fixes.config

```diff
@@ -20774,6 +20774,9 @@ CSS
 #maincontainer {
     color: var(--darkreader-neutral-text) !important;
 }
+select {
```

Comment:
I know you tried to make nice fix but we try not use `url("data:image"` in sitefixes.
Second thing... As Dark Reader can use light or dark scheme for applying it's changes to the site then `fill:%23bbb` should be changed to something more neutral like `fill:%23888`